certified-connectors/Softools/apiDefinition.swagger.json

```diff
@@ -0,0 +1,879 @@
+            "x-ms-summary": "The tenant you want to connect too",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Response also includes all record data with field identifier as key",
+            "schema": {
+              "$ref": "#/definitions/Message"
+            }
+          },
+          "401": {
+            "description": "",
+            "examples": {
+              "application/json": "{\n  \"Message\": \"Authorization has been denied for this request.\"\n}"
+            }
+          }
+        }
+      }
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.softools.net"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.softools.net/privacy-policy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Business Intelligence;Data"
+    }
+  ],
+  "definitions": {
+    "Message": {
+      "type": "object",
+      "properties": {
+        "Message": {
+          "type": "string"
+        }
+      },
+      "example": {
+        "Message": "string"
+      }
+    },
+    "Records": {
+      "type": "object",
+      "example": "[\n  {\n    \"TextFieldExample\": \"\",\n    \"DateFieldExample\": null,\n    \"RangeFieldExample\": \"0\",\n    \"MultiStateHarveyBallFieldExample\": \"\",\n    \"MultiStateRAGFieldExample\": \"\",\n    \"BitFieldExample\": false,\n    \"NumberFieldExample\": 0,\n    \"SingleSelectionExample\": \"Item A\",\n    \"ListFieldExample\": {\n      \"changes\": { },\n      \"added\": [ ],\n      \"removed\": [ ]\n    }\n  },\n  {\n    \"TextFieldExample\": \"\",\n    \"DateFieldExample\": null,\n    \"RangeFieldExample\": \"0\",\n    \"MultiStateHarveyBallFieldExample\": \"\",\n    \"MultiStateRAGFieldExample\": \"\",\n    \"BitFieldExample\": false,\n    \"NumberFieldExample \": 0,\n    \"SingleSelectionExample\": \"Item A\",\n    \"ListFieldExample\": {\n      \"changes\": { },\n      \"added\": [ ],\n      \"removed\": [ ]\n    }\n  }\n]"
+    },
+    "basic-fields-response": {
+      "type": "object",
+      "properties": {
+        "identifier": {
+          "type": "string"
+        },
+        "Type": {
+          "type": "integer"
+        }
+      }
+    },
+    "List-record-response": {
+      "type": "array",
+      "items": {},
+      "example": [{
+        "_id": "id",
+        "QuickFilterSearchText": "",
+        "AccessRights": [
+          {
+            "_id": {
+              "$oid": "accessId"
+            },
+            "AccessType": 1,
+            "SourceType": 2,
+            "ActorType": 3,
+            "ActorId": "actorId",
+            "ParentId": null,
+            "Permissive": true
+          }
+        ],
+        "Id": "id",
+        "EditableAccessForUser": true,
+        "Tenant": "",
```

Comment:
@thduttonuk Empty property values can be removed until unless you have a specific reason to keep them.